README.md

```diff
@@ -1,2 +1,125 @@
 # neofs-sdk-go
-Go implementation of NeoFS SDK
+Go implementation of NeoFS SDK. It contains high-level version-independent wrappers
+for structures from [neofs-api-go](https://github.com/nspcc-dev/neofs-api-go) as well as
+helper functions for simplifying node/dApp implementations.
+
+## Repository structure
+
+### accounting
+Contains fixed-point `Decimal` type for performing balance calculations.
+
+### acl
+Contains set of constants to cover common use-cases when working with NeoFS.
+See [ACL speciifcation](https://github.com/nspcc-dev/neofs-spec/blob/master/01-arch/07-acl.md) for details.
+
+### eacl
+Contains Extended ACL types for fine-grained access control.
+There is also a reference implementation of checking algorithm which is used in NeoFS node.
+
+### checksum
+Contains `Checksum` type encapsulating checksum as well as it's kind.
+Currently Sha256 and [Tillich-Zemor hashsum](https://github.com/nspcc-dev/tzhash) are in use.
+
+### owner
+`owner.ID` type represents single account interacting with NeoFS. In v2 version of protocol
+it is just raw bytes behing [base58-encoded address](https://docs.neo.org/docs/en-us/basic/concept/wallets.html#address)
+in Neo blockchain. Note that for historical reasons it contains
+version prefix and checksum in addition to script-hash.
+
+### signature
+`signature.Signature` type consists of public key and signature payload.
+In the future, it will also contain kind for handling different signature
+algorithms, see [nspcc-dev/neofs-api#55](https://github.com/nspcc-dev/neofs-api/issues/55) for
+the corresponding issue. Also, see `util` package for signature-related helpers.
+
+### token
+Contains [Bearer token](https://oauth.net/2/bearer-tokens/) type with several
```

Comment:
BearerToken in NeoFS is a little bit different thing, let's not reference OAuth here to avoid confusion.